tests/evaluation/dataset/test_studio_data_repository.py

```diff
@@ -55,28 +63,38 @@ def test_create_dataset(
     ]
 
     dataset = studio_dataset_repository.create_dataset(
-        examples=examples, dataset_name="Dataset 1", labels={"label"}, metadata={}
+        examples=examples,
+        dataset_name="Dataset 1",
+        labels={"label"},
+        metadata={},
     )
 
     assert isinstance(dataset, Dataset)
-    assert dataset.id == "dataset1"
+    assert dataset.id == expected_dataset_id
     assert dataset.name == "Dataset 1"
     assert dataset.labels == {"label"}
     assert dataset.metadata == {}
 
-    mock_data_client.create_dataset.assert_called_once_with(
-        repository_id="repo1",
-        dataset=DatasetCreate.model_validate(
-            {
-                "source_data": b'{"input":{"data":"input1"},"expected_output":{"data":"output1"},"id":"example1","metadata":null}\n{"input":{"data":"input2"},"expected_output":{"data":"output2"},"id":"example2","metadata":null}',
-                "labels": ["label"],
-                "name": "Dataset 1",
-                "total_datapoints": 2,
-                "metadata": {},
-            }
-        ),
+    actual_call = studio_dataset_repository.studio_client.submit_dataset.call_args  # type: ignore
+    submitted_dataset = actual_call[1]["dataset"]
+    submitted_examples = actual_call[1]["examples"]
+
+    expected_dataset = StudioDataset.model_validate(
+        {
+            "labels": ["label"],
+            "name": "Dataset 1",
+            "metadata": {},
+        }
```

Comment:
i think it would be better to just initialize it instead of doing it via model.validate
. We could also pull these values from the top call to make sure they are equal

Reply:
Refactor into variable